guides/Other Subjects/RichMenu.md

```diff
@@ -42,6 +37,13 @@ module.exports = class extends Command {
 	}
 
 	init() {
+		this.menu = new RichMenu(new MessageEmbed()
+			.setColor(0x673AB7)
+			.setAuthor(this.client.user.username, this.client.user.avatarURL())
```

Comment:
Just one detail that we missed in the old code:
```suggestion
			.setAuthor(this.client.user.username, this.client.user.displayAvatarURL())
```

Reply:
Done!